src/test/java/steps/NavigationStep.java

```diff
@@ -0,0 +1,33 @@
+package steps;
+
+import org.openqa.selenium.WebElement;
+import pages.MainPage;
+import pages.NewsPage;
+
+public class NavigationStep {
```

Comment:
What's the difference between NavigationStep and NavigationResultSteps?

Reply:
NavigaitonResultStep used to work with the results obtained from NavigationStep. The idea is to differentiate the methods used for navigation tests for convenience.